src/client/kotlin/tech/thatgravyboat/skyblockapi/api/data/SkyBlockStat.kt

```diff
@@ -12,17 +12,80 @@ enum class SkyBlockStat(
     val color: Int,
     name: String? = null,
 ) {
+    // Combat stats
+
     HEALTH('❤', TextColor.RED),
     DEFENSE('❈', TextColor.GREEN),
     STRENGTH('❁', TextColor.RED),
     INTELLIGENCE('✎', TextColor.AQUA),
-    SPEED('✦', TextColor.WHITE),
     CRIT_DAMAGE('☠', TextColor.BLUE),
     CRIT_CHANCE('☣', TextColor.BLUE),
-    ATTACK_SPEED('⚔', TextColor.YELLOW),
-    // TODO: add more stats,
+    BONUS_ATTACK_SPEED('⚔', TextColor.YELLOW),
```

Comment:
Need to be `ATTACK_SPEED` or it wont be compatible with older versions

Reply:
should I deprecate `ATTACK_SPEED` and add `BONUS_ATTACK_SPEED` or just rename it again?

Reply:
I think just renaming is fine